index.html

```diff
@@ -5,9 +5,71 @@
 
     <link href="styles.css" rel="stylesheet" />
 
-    <title>Rocket Academy</title>
+    <title>Hi I'm Patrick</title>
   </head>
   <body>
-    <h1>Hello, world!</h1>
+    <div class="header">
+      <img class="profile-pic" src="image/headshot.jpg" />
+      <div class="navbar">
+        <div class="nav"><a href="#skills">Skills</a></div>
+        <div class="nav"><a href="#projects">Projects</a></div>
+        <div class="nav"><a href="#links">My Links</a></div>
+      </div>
+    </div>
+    <div id="about-me">
+      <div class="introduction">
+        Hi I'm Patrick, an aspiring software developer, welcome to my portfolio
+      </div>
+    </div>
+    <div id="skills">
+      <div class="skills-container">
+        <div class="skill-header">Soft Skills</div>
+        <div>
+          <ol>
+            <li>Empathy</li>
+            <li>Communication</li>
+            <li>Leadership</li>
+          </ol>
+        </div>
+      </div>
+      <div class="skills-container">
```

Comment:
nice class naming. For more about css naming conventions, can use BEM possibly: https://en.bem.info/methodology/naming-convention/